ports/servoshell/Cargo.toml

```diff
@@ -70,6 +70,7 @@ libc = { workspace = true }
 libservo = { path = "../../components/servo", features = ["background_hang_monitor", "bluetooth", "testbinding"] }
 log = { workspace = true }
 mime_guess = { workspace = true }
+muda = "0.17.1"
```

Comment:
I would prefer not to add another dependency, especially one that adds dependencies on platform-specific toolkits. Please implement these menus as egui context menus.